pypeit/scripts/show_2dspec.py

```diff
@@ -95,6 +95,8 @@ def main(args):
 
     # Object traces from spec1d file
     spec1d_file = args.file.replace('spec2d', 'spec1d')
+    if args.file[-2:] == 'gz':
+        spec1d_file = spec1d_file[:-3]
```

Comment:
At the moment, neither are being compressed.  I added this 
for when I was compressing `spec2d` and left it in as it does
no harm (in case we ever start doing so).